openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -865,6 +870,44 @@ def test_virtual_sites_no_reassign(benzene_system, toluene_system,
             dag_unit.run(dry=True)
 
 
+def test_dodecahdron_ligand_box(benzene_system, toluene_system,
+                                benzene_to_toluene_mapping, tmpdir):
+    """
+    Test that a hybrid system with a dodechadron is built properly.
+    """
+    settings = openmm_rfe.RelativeHybridTopologyProtocol.default_settings()
+    settings.solvation_settings.solvent_padding = 1.5 * unit.nanometer
+    settings.solvation_settings.box_shape = 'dodecahedron'
```

Comment:
Should this become our new recommendation? (dodecahedron boxes)

Reply:
Long term yes, but we'll need to do some performance benchmarks.